client/src/components/global/Date.js

```diff
@@ -0,0 +1,49 @@
+import React, { Component, PropTypes } from 'react';
+import format from 'date-fns/format';
+import parse from 'date-fns/parse';
+
+export default class Date extends Component {
+  static propTypes = {
+    prefix: PropTypes.string,
+    format: PropTypes.string,
+    date: PropTypes.oneOfType([
+      PropTypes.string,
+      PropTypes.object
+    ])
+  };
+
+  formatDate() {
+    let date = null;
+    if (this.props.date) {
+      if (this.props.date instanceof Object) {
+        date = (
+          format(this.props.date, this.props.format)
+        );
+      } else {
+        date = (
+          format(parse(this.props.date), this.props.format)
+        );
+      }
+    }
+
+    return date;
+  }
+
+  formatString(date) {
+    let output = null;
+    if (this.props.prefix) {
+      output = this.props.prefix + ' ' + date;
+    } else {
+      output = date;
+    }
+
+    return output;
+  }
+
+  render() {
+    const date = this.formatDate();
+    return (
+      <span>{ this.formatString(date) }</span>
```

Comment:
We typically don't put spaces around the method call here. I might also wrap this up in a small method.
```
value() {
  return this.formatString(this.formatDate(this.props.date));
}

render() {
  return <span>{this.value()}</span>
}
```